app/src/main/java/com/ecosystem/kin/app/MainActivity.java

```diff
@@ -57,18 +66,21 @@
 	protected void onCreate(Bundle savedInstanceState) {
 		super.onCreate(savedInstanceState);
 		setContentView(R.layout.activity_main);
+		userID = SignInRepo.getUserId(getApplicationContext());
```

Comment:
userID isn not required within the onCreate logic. Maybe just provide getUserID method ?

Reply:
not really important